frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -669,32 +669,86 @@ export interface OwnedPodsSectionProps {
 export function OwnedPodsSection(props: OwnedPodsSectionProps) {
   const { resource, hideColumns } = props;
 
-  const [pods, error] = Pod.useList();
```

Comment:
(Not related to this line)
Please prefix the commit name with `frontend: ` if it only touched the frontend.

Reply:
Aha, fixed it.